packages/e2e-tests/kit-node/__tests__/kit.spec.ts

```diff
@@ -16,6 +16,7 @@ import {
 
 import glob from 'tiny-glob';
 import path from 'path';
+import { describe, expect } from 'vitest';
```

Comment:
Should we also import `it`? 
```suggestion
import { describe, it, expect } from 'vitest';
```